pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -211,7 +237,7 @@ private boolean createDictionaryForColumn(ColumnIndexCreationInfo info, SegmentG
       FieldSpec spec) {
     String column = spec.getName();
 
-    if (config.getRawIndexCreationColumns().contains(column)) {
+    if (config.getRawIndexCreationColumns().contains(column) || config.getRawIndexCompressionType().containsKey(column)) {
```

Comment:
Is there any case where we have columns not specified in `RawIndexCreationColumns` but specified in`RawIndexCompressionType`? I think that it makes more sense to explicitly check `RawIndexCreationColumns` only.

Reply:
Ideally, we should only have one config, but seems like  the RawIndexCompressionTypeMap was introduced a while back. And we cannot simply merge the two due to backward compatibility. So for now, I made the code relaxed enough to handle both.